README.md

```diff
@@ -22,6 +22,14 @@ If the pipeline is backed by Orchestra, the latest published version will run. I
 
 (Optional) The environment name or ID to run the pipeline in. If not specified, the default environment will be used
 
+### `task_ids`
+
+(Optional) If the user wants to run a subset of tasks in the pipeline, they can specify the IDs of the tasks to run. No task group ids are supported.
+
+### `continue_downstream_run`
+
+(Optional) If the user wants to continue a downstream run after running specific task IDs just set this to true.
```

Comment:
same comment as above re referencing 'the user'

also would remove the word 'just' (not needed, sentence reads the same without it)